cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -196,7 +196,8 @@ func (command *Evaluate) Initialize(args []string) (evaluationContext *evaluate.
 				command.logger.Panicf("ERROR: unknown custom provider %q for model %q", providerID, model)
 			}
 
-			modelProvider.AddModel(llm.NewModel(modelProvider, model))
+			modelName := strings.Split(model, provider.ProviderModelSeparator)
+			modelProvider.AddModel(llm.NewNamedModelWithCost(modelProvider, model, modelName[len(modelName)-1], 0))
```

Comment:
Can use https://pkg.go.dev/strings#SplitN to get just the model name.

`strings.Split(model, provider.ProviderModelSeparator, 1)[1]`

Reply:
Wait why was this not resolved... I guess it should've been `strings.SplitN(model, provider.ProviderModelSeparator, 2)[1]`.